README.md

```diff
@@ -200,18 +200,45 @@ The tool discovers checks first, then appends any `additional_checks` are concat
 
 ### Excluding specific jobs
 
-The easiest way to exclude a single job is via the `name` parameter:
+You can exclude specific jobs by using their names:
 
 ```json
 {
   "exclude": [
     {
-      "name": "PHPUnit on PHP 8.0 with latest dependencies"
+      "name": "PHPUnit"
```

Comment:
Will the old exclusion via exact name match still work? I saw no changes in tests, which seems to hint at that, but I'm not 100% sure that's covered :D

Reply:
yes that would work but something is still not properly working and thus have to re-check here. No need to have this as a blocker for v2. I can adapt.

Reply:
Nono, I mean that you can merge and release `1.23.x` whenever: I'll deal with `2.0.x` over the weekend, if I can.